src/core/Track.cpp

```diff
@@ -1864,7 +1964,7 @@ bool TrackContentWidget::pasteSelection( MidiTime tcoPos, const QMimeData * md,
 		if( wasSelection )
 		{
 			tco->selectViewOnCreate( true );
-		}
+    	}
```

Comment:
Some spaces have sneaked in at the start of the line here - change back to tabs?

Reply:
Yup, done